src/test/java/IBHW18.java

```diff
@@ -0,0 +1,31 @@
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+import org.testng.annotations.Test;
+
+public class IBHW18 extends BaseTest{
+
+    @Test
+    public void playASongTest() {
+        login("ionut.burtoiu@testpro.io", "te$t$tudent");
+        playSong();
+        isSongPlaying();
+    }
+
+    public void playSong() {
+        WebElement playNextButton = driver.findElement(By.xpath("//i[@data-testid='play-next-btn']"));
+        WebElement playButton = driver.findElement(By.xpath("//span[@data-testid='play-btn']"));
+
+        playNextButton.click();
+        playButton.click();
+    }
+
+    public void isSongPlaying() {
+        WebElement soundBar = driver.findElement(By.xpath("//div[@data-testid='sound-bar-play']"));
+        Assert.assertTrue(soundBar.isDisplayed());
+    }
+
+
+
+
```

Comment:
**Please delete the empty line!** An Empty Line is a "Code Smell"
The short answer is, _They’re more an annoyance than a problem.They won’t actually break anything—which is why your code continues to run—but they make it awkward to read and change your code, especially if you’re dealing with multiple line breaks per line of real code. They also add just a tiny bit of size to your files, which is nice to reduce._